src/actions/SignupActions.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import request from 'superagent'
+import LoginConstants from '../constants/LoginConstants.js'
+
+export const signupSuccess = () => ({
+  type: LoginConstants.SIGNUP_SUCCESS,
+  success: true,
+  description: 'Check your email to validate your account',
+})
+
+export const signupFailure = () => ({
+  type: LoginConstants.SIGNUP_FAILURE,
+  success: false,
+  description: 'Something went wrong',
+})
+
+export const signupUser = (
+  firstName: string,
+  lastName: string,
+  email: string,
+  password: string
+) => (dispatch: Function) => {
+  request.post('http://localhost:5000/signup')
```

Comment:
Is this port your oberon server? It doesn't match the port that npm start runs this application